vespamalloc/src/tests/test1/new_test.cpp

```diff
@@ -101,9 +101,11 @@ TEST("verify realloarray") {
     void *arr2 = reallocarray(arr, 800, 5);
     EXPECT_NOT_EQUAL(arr, arr2);
     EXPECT_NOT_EQUAL(nullptr, arr2);
+    EXPECT_NOT_EQUAL(ENOMEM, errno);
```

Comment:
Might need to clear `errno` right before calling `reallocarray` to ensure that it is not accidentally `ENOMEM`.    `errno` should also be read right afterwards and stored in another variable, since it can potentially be changed as a side effects of other calls (e.g. `EXPECT_NOT_EQUAL` on the lines before).